driver/src/main/java/oracle/nosql/driver/util/BinaryProtocol.java

```diff
@@ -187,6 +187,8 @@ public static boolean isTenantOperation(OpCode op) {
     /* added in V4 */
     public static final int TABLE_NOT_READY = 26;
     public static final int UNSUPPORTED_QUERY_VERSION = 27;
+    /* added in V5 */
```

Comment:
It seems we don't have serial version V5, the current version is V4 that is for NSON protocol I think. So maybe no need this comment?